server/settings.py

```diff
@@ -179,3 +179,6 @@
 }
 
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
+
+# Add trusted origins
+CSRF_TRUSTED_ORIGINS = [f"http://localhost:3000"]
```

Comment:
```suggestion
# React client is on 3000, while django admin auth is on 8000
if os.getenv("DEPLOY_ENV") == "development":
   CSRF_TRUSTED_ORIGINS = ["http://localhost:3000", "http://127.0.0.1:3000"]
```

We should strictly enable just for dev instances only.

The CSRF_TRUSTED_ORIGINS  is already self explanatory, would be great if we explained why in the comments instead.